### PR TITLE
[GECO-65] show version number in footer

### PR DESCRIPTION
**Pull request by snilapwa**

### CODE REVIEW

File: cepheus/pom.xml [hunk 1]
```diff
@@ -16,6 +16,8 @@
         <geco.cepheus.version>0.2.2-SNAPSHOT</geco.cepheus.version>
 
         <admin.password>admin</admin.password>
+        <pullrequest></pullrequest>
+        <buildNumber></buildNumber>
```
Comment (jdamerow) [inline cepheus/pom.xml]: same here. buildnumber plugin is missing.

File: cepheus/pom.xml [hunk 2]
```diff
@@ -324,6 +326,29 @@
...
+                    <doCheck>false</doCheck>
+                    <doUpdate>false</doUpdate>
+                    <revisionOnScmFailure></revisionOnScmFailure>
+                    <items>
+                        <item>buildNumber\d*</item>
+                        <item>timestamp</item>
+                    </items>
+                    <format>#{0} ({1,date,yyyy-MM-dd HH:mm:ss})</format>
+                </configuration>
+            </plugin>
```
Comment (jdamerow) [inline cepheus/pom.xml]: have you tried this, is that actually generating build numbers? I believe it needs a scm definition.

Comment (snilapwa) [inline cepheus/pom.xml]: updated with scm definition

**Comment by snilapwa** [comment]
updated gitignore

**Pull request merged**